openmc/plotter.py

```diff
@@ -438,6 +439,19 @@ def _calculate_cexs_nuclide(this, types, temperature=294., sab_name=None,
                             [sab_sum, nuc[mt].xs[nucT]],
                             [sab_Emax])
                         funcs.append(pw_funcs)
+                    elif ncrystal_cfg:
+                        import NCrystal
+                        nc_scatter = NCrystal.createScatter(ncrystal_cfg)
+                        nc_func = nc_scatter.crossSectionNonOriented
+                        nc_emax = 5 # eV # this should be obtained from NCRYSTAL_MAX_ENERGY
```

Comment:
I think there was some discussion about having the max energy be configurable, but I forget where we ended up on this. @tkittel Do you recall? In any event, I'm fine with the hard-coded value for now.